feddlib/core/FE/FiniteElement.hpp

```diff
@@ -120,6 +142,13 @@ class FiniteElement {
 	GO predecessorElement_ = -1;
 	LO refinementEdge_=-1;
 	vec_LO_Type markedEdges_;
+
+    // Defined variables for boundary elements
+    vec_dbl_Type surfaceNormal;                          // As we do not have dim here as variable we cannot initialize the vector 
+    bool neumannBC_element = false;                      // Check if the element is an surface element where we want to add an additional contribution - only for boundary elements where we want to add surface integral
+    vec_dbl_Type neumannBC_QuadratureWeightsRef;         // These are the Quadrature weights in terms of quadrature points defined in reference coordinate system so here [0 1]
+    vec2D_dbl_Type neumannBC__QuadraturePointsPhy;       // 2D: quadrature points on line, 3D: quadrature points in plane, have to be mapped on reference element with mapping in specificAssemblyClasses
```

Comment:
Naming convention and Phy <--> Phi

Reply:
With Phy  I meant Physical coordinate system but I think global vs reference is better

Reply:
Ah perfect. That's less confusing name. Sorry, I didn't look into the functionality in detail.